__tests__/hello.po

```diff
@@ -0,0 +1,11 @@
+msgid "hello"
+msgid_plural "hellos"
+msgstr[0] ""
+msgstr[1] ""
+
+
+msgctxt "homepage"
+msgid "hi"
```

Comment:
both of these `msgid` are not great, esp when used with in plural.  I think creating a `animal.po` with `cat`/`cats` & `dog`/`dogs` would be better.  I'll do that once the overall structure of this feature is reviewed.